www/js/draw-model.js

```diff
@@ -479,6 +549,24 @@ define([
 		updateNodeConnections: function(from, to){
 			this.detachConnections(to);
 			this.setConnections(from, to);
+		},
+		getStatus: function(node){
+			var nodeStatus = node.status;
+			var nodeID = node.ID;
+			var statusClassMap = {
+				"demo" : "fa-minus",
+				"incorrect" : "fa-times",
+				"correct" : "fa-check",
+				"perfect" : "fa-star"
+			}
+			nodeStatus = this._model.getCorrectness?this._model.getCorrectness(nodeID):"";
+			//Check for perfect node
+			if(this._model.isComplete(nodeID) && this._model.getAssistanceScore(nodeID) === 0 && this._model.getCorrectness(nodeID) === "correct"){
+				nodeStatus = "perfect";
+			}
```

Comment:
this is what I mean, these decisions are to be taken by PM and it should tell that nodeStatus is perfect. so show whatever u have for that.